src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Automation</PackageId>
     <Description>Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.</Description>
-    <Version>3.1.0-preview</Version>
+    <Version>3.2.0-preview</Version>
```

Comment:
Please update the `PackageReleaseNotes` 
Also, please make it descriptive similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj#L13)

Reply:
Done.